Day-I-Social-Card/social-card/src/components/CardComponents/CardContainer.js

```diff
@@ -1,2 +1,17 @@
 import React from 'react';
 import './Card.css';
+
+import CardBanner from './CardBanner';
+import CardContent from './CardContent';
+
+function CardContainer(props) {
+    return (
+        <div class='card-container'>
+            <CardBanner />
+
+            <CardContent />
```

Comment:
I think the idea for this project is passing props. So you would write line 12 like `<CardContent h2Prop="Get Started with React" />`.